lib/tests_coordination.sh

```diff
@@ -138,26 +138,7 @@ parse_check_process() {
                 > "$TEST_CONTEXT/tests/$test_id.json"
         }
 
-        # For not-the-main-test-serie, we only consider testing the install and
-        # upgrade from previous commits
-        if [[ "$test_serie_id" != "1" ]]
-        then
-            test_serie=${tests_serie//;; }
```

Comment:
Uuuh, shouldnt we at least keep this line ? Otherwise `test_serie` will always be equal to `default` even if it's not the default one, and that probably break stuff somewhere o_O

Reply:
I really don't know, I wasn't able to understand those lines of code :/